src/view_pos_transform.hpp

```diff
@@ -0,0 +1,178 @@
+// ==========================================================================
+//                                  lambda
+// ==========================================================================
+// Copyright (c) 2013-2019, Hannes Hauswedell <h2 @ fsfe.org>
+// Copyright (c) 2016-2020, Knut Reinert and Freie Universität Berlin
+// All rights reserved.
+//
+// This file is part of Lambda.
+//
+// Lambda is Free Software: you can redistribute it and/or modify it
+// under the terms found in the LICENSE[.md|.rst] file distributed
+// together with this file.
+//
+// Lambda is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+//
+// ==========================================================================
+// view_pos_transform.hpp
+// ==========================================================================
+
+#include <seqan3/range/views/all.hpp>
```

Comment:
Please use the correct headers here and not `all.hpp`! I did that in my example because I was lazy, but it needlessly lengthens compile-time.